builds/e2e/templates/unlock-test-agents.yaml

```diff
@@ -31,10 +31,10 @@ steps:
         --request PUT "https://msazure.visualstudio.com/_apis/distributedtask/pools/$POOL_ID/agents/$agentId/usercapabilities" \
         -H "Content-Type:application/json" \
         -H "Accept: application/json;api-version=5.0;" \
-        --max-time 10 \
-        --retry 5 \
+        --max-time 15 \
```

Comment:
How is this different from the script below?

Reply:
For some reason, dup comments are showing. This script is for unlocking where the other is for locking. The way this came about is both of these were bash logic in yaml files, but the locking became complicated so we extracted it out. We could merge these together, but before I would do that I would rather just port the whole agent lock/unlock logic to dotnet. One of the valuable lessons I learned from this experience is to avoid bash when the logic gets even a bit non-trivial.

 
- Both of these were bash logic in yaml files
- Locking logic became compli